apps/right-to-rent-check/acceptance/features/tenant-delete-last.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+const _ = require('lodash');
+const steps = require('../../');
+
+Feature('Deleting the last tenant');
+
+Before((
+  I,
+  tenantAddAnotherPage
+) => {
+  I.visitPage(tenantAddAnotherPage, steps);
+  I.seeNumberOfElements('.tenant-details', 1);
+});
+
+Scenario('I see all fields empty on the the tenant-details page', (
+  I,
+  tenantAddAnotherPage,
+  tenantDetailsPage,
+  tenantAdditionalDetailsPage
+) => {
+  I.click('.tenant-details:first-of-type tr:nth-child(7) > td:nth-child(4) > a');
```

Comment:
I'm assuming this is the Delete button, is there any way of making that clear e.g. maybe a comment or adding it to the associated page file?  Also I think this feature could have the kind of explanation you talked about before, 'Given I delete the last tenant on the tenant-add-another page', Scenario 'I see all fields empty on the tenant-details-page